playbooks/get_pod_status.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+OUTPUT=/tmp/pod_status
+
+python -c "import kubernetes" > /dev/null
+
+if [[ $? -gt 0 ]]; then
+  pip install kubernetes --user
+fi
+
+# k8s_status=$(python /vagrant/pods.py)
+python /vagrant/pods.py > ${OUTPUT} 2>&1
+
+COUNTER=0
+while [[ $(cat ${OUTPUT} | grep "Connection refused") ]] && [[ $COUNTER -lt 15 ]]; do
+  # echo "Waiting for kubernetes to be ready - attempt # ${COUNTER}"
+  echo "Waiting for kubernetes to be ready - attempt # $((COUNTER+1))"
+  sleep 30
+  # k8s_status=$(python /vagrant/pods.py)
+  python /vagrant/pods.py > ${OUTPUT} 2>&1
+  let COUNTER=COUNTER+1
+done
+
+python /vagrant/pods.py
```

Comment:
Why are you calling the python script from a bash script? You can do everything you are doing in this script in python adn the whole implementation is going to be more clean.

Reply:
At first I was having difficulties installing and importing pip modules within the python script. I moved the installation of python modules to the vagrant shell provider and the handling of the exception to the python script.

Reply:
Could you check again if you can improve this/ It can be done moving all the logic in the python script.

Reply:
Sure! I'll get back to you once I get it right

Reply:
Managed to handle the installation of pip modules within the python scripts thanks to the hints in this answer: https://stackoverflow.com/questions/12332975/installing-python-module-within-code